automatic/waterfox/waterfox.nuspec

```diff
@@ -4,8 +4,8 @@
   <metadata>
     <id>waterfox</id>
     <version>50.1.0</version>
-    <packageSourceUrl>https://github.com/AdmiringWorm/chocolatey-packages/tree/master/waterfox</packageSourceUrl>
```

Comment:
thanks for fixing this, was originally planning to add it to my repository.
Guess I forgot to change it.